core/statistics.c

```diff
@@ -174,6 +191,26 @@ void calculate_stats_summary(struct stats_summary *out, bool selected_only)
 		process_dive(dp, &(out->stats_by_type[dp->dc.divemode + 1]));
 		out->stats_by_type[dp->dc.divemode + 1].selection_size++;
 
+		/* stats_by_depth[0] is all the dives combined */
+		out->stats_by_depth[0].selection_size++;
+		process_dive(dp, &(out->stats_by_depth[0]));
+
+		d_idx = dp->maxdepth.mm / (STATS_DEPTH_BUCKET * 1000);
+		if (d_idx < 0) { d_idx = 0; }
```

Comment:
Here and below:
Coding style: no braces for single statement `if`, `while`, `for`.

Reply:
... and with very rare exceptions, we prefer the conditionally executed block on the next line.
So this should be
```
if (d_idx < 0)
        d_idx = 0;
```